README.md

```diff
@@ -123,9 +123,22 @@ Additionally you can configure many different types of custom Jira field, which
 > Depending on Jira response time i had successfully listed 1000 issues but sometimes only about >850.
 
 
+### Refresh IssueTable
+“Add-ons" > “Project Aid for Jira” > "Refresh IssueTable"
+
+YouTube: [Watch this feature on YouTube](https://youtu.be/9TWuvRhx7Fw)
```

Comment:
Cool that you have made a video :) 
I went to watch it to understand the functionality but I get a youtube error saying its unavailable (I expect because you havent made it public yet) - just an FYI in case this is not expected.

Reply:
Yes, its not made public yet.